PyExpLabSys/drivers/keithley_2000.py

```diff
@@ -0,0 +1,121 @@
+""" Simple driver for Keithley 2000 DMM """
+from PyExpLabSys.drivers.scpi import SCPI
+
+
+class Keithley2000(SCPI):
+    """
+    Simple driver for Keithley 2000 DMM
+    """
+
+    def __init__(self, interface, hostname='', device='',
+                 baudrate=9600, gpib_address=None):
+        if interface == 'serial':
+            SCPI.__init__(self, interface=interface, device=device,
+                          baudrate=baudrate, line_ending='\n')
+            self.comm_dev.timeout = 2
+            self.comm_dev.rtscts = False
+            self.comm_dev.xonxoff = False
+        if interface == 'gpib':
+            SCPI.__init__(self, interface=interface, gpib_address=gpib_address)
+
+    def set_bandwith(self, measurement='voltage:ac', bandwidth=None):
+        scpi_cmd = 'SENSE:{}:DETector:BANDwidth'.format(measurement)
+        if bandwidth is not None:
+            DMM.scpi_comm(scpi_cmd + ' {}'.format(bandwidth))
+        value_raw = DMM.scpi_comm(scpi_cmd + '?')
+        value = float(value_raw)
+        return value
+
+    def set_range(self, value: float):
+        """
+        Set the measurement range of the device, 0 will indicate auto-range
+        """
+        if value > 1000:
+            value = 1000
+        if value < 0:
+            value = 0
+        if value == 0:
+            self.scpi_comm(':SENSE:VOLT:DC:RANGE:AUTO ON')
+            self.scpi_comm(':SENSE:VOLT:AC:RANGE:AUTO ON')
+        else:
+            self.scpi_comm(':SENSE:VOLT:DC:RANGE {:.5f}'.format(value))
+            self.scpi_comm(':SENSE:VOLT:AC:RANGE {:.5f}'.format(value))
+
+        actual_range_raw = self.scpi_comm(':SENSE:VOLTAGE:AC:RANGE?')
+        actual_range = float(actual_range_raw)
+        return actual_range
+
+    def set_integration_time(self, nplc: float = None):
+        """
+        Set the measurement integration time
+        """
+        if nplc is not None:
+            if nplc < 0.01:
+                nplc = 0.01
+            if nplc > 60:
+                nplc = 60
+            self.scpi_comm('SENSE:VOLTAGE:AC:NPLCYCLES {}'.format(nplc))
+            # self.scpi_comm('SENSE:VOLTAGE:DC:NPLCYCLES {}'.format(nplc))
```

Comment:
Old code can be fetched from git, so I suggest out with this

Reply:
But finding it when you need it takes 10^7 times longer than just re-enabling this.